framework/tlvf/tlvf.py

```diff
@@ -644,6 +646,41 @@ def addClassParam(self, obj_meta, param_name, param_type, param_type_info, param
             if TypeInfo(param_type).type == TypeInfo.CLASS:                
                 lines_h.append("%s *m_%s = nullptr;" % (param_type, param_name))
                 lines_h.append("std::shared_ptr<%s> m_%s_ptr = nullptr;" % (param_type, param_name))
+                lines_h.append("bool m_%s_init = false;" % (param_name))
+                
+                #Add a get method in the H file
```

Comment:
Should go into a function with documentation explaining all we are doing here and why. Its about time that tlvf gets some good comments.